Jint/Engine.Ast.cs

```diff
@@ -22,12 +28,13 @@ public partial class Engine
     public static Script PrepareScript(string script, string? source = null, bool strict = false)
     {
         var astAnalyzer = new AstAnalyzer(new ScriptPreparationOptions());
-        var options = ParserOptions.Default with
+        var options = s_defaultParserOptions with
```

Comment:
Why is this name so ugly? Or do you fancy this style?

Reply:
Actually, it came from Jint's .editorconfig: https://github.com/sebastienros/jint/blob/23a2a2587750a98cd55e4697e4da8270bcf460db/.editorconfig#L114

From what I see when looking around the codebase, this rule is not really followed. :) So, I'll change it to be consistent with the rest.